business_requirement/controllers/portal.py

```diff
@@ -12,11 +12,13 @@ class CustomerPortal(CustomerPortal):
 
     def _prepare_portal_layout_values(self):
         values = super(CustomerPortal, self)._prepare_portal_layout_values()
-        br_count = request.env['business.requirement'].search_count(
-            self._prepare_br_base_domain())
-        values.update({
-            'business_requirement_count': br_count,
-        })
+        BRObj = request.env['business.requirement']
```

Comment:
snake case for variables 
```suggestion
        br_obj = request.env['business.requirement']
```

Reply:
User either `BR` or `br_obj`, but not a mix

Reply:
@victormmtorres I all times use UpperCamelCase for this kind of vars... You can see it in other parts of Odoo code.

Reply:
@Tardo, `BrObj` would be UpperCamelCase, but not `BRObj`.